remover.py

```diff
@@ -235,7 +243,24 @@ def remove_images():
 
     print("Job is done")
     root.quit()
+def remove_images2():
+    writer          = PdfEnhancedFileWriter()
+    # output_filename = asksaveasfilename(filetypes = (('PDF File', '*.pdf'), ('All Files','*.*')))

```

Comment:
I'm not sure what's the reason for the second remove_images?
if you split cases- give it an indicative name,
better if you are able to find a way to make some condition testing that will automatically tell which one to choose.

if you are struggling- send me an example pdf of it, we can think together.

Reply:
One of the major problems, are tests don't have a uniform types. Or canonical forms.